README.md

```diff
@@ -5,4 +5,6 @@ This is the repository used in Plone 6 demo in 6.demo.plone.org.
 You can find the backend in the `/api` folder, and the frontend in the root of the repo.
 You can find also convenience scripts in the supplied `Makefile`.
 
+For convenience, the deployment Ansible playbooks are included in `ansible` folder. They install the entire required dependencies and infrastructure, and the services as well.
+
 For any questions, please contact the @plone/ai-team
```

Comment:
Although not part of this PR, I have no idea how to contact `@plone/ai-team`. Is that an email address, Twitter handle, other? Please be explicit.

Reply:
It's the GitHub handler, @ericof there's a mail for the aiteam?